src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_keychain.c

```diff
@@ -168,8 +170,9 @@ static OSStatus DeleteInKeychain(CFTypeRef needle, SecKeychainRef haystack)
     if (query == NULL)
         return errSecAllocate;
 
+    const void *constKeychain = haystack;
     CFArrayRef searchList = CFArrayCreate(
-        NULL, (void**)(&haystack), 1, &kCFTypeArrayCallBacks);
+        NULL, &constKeychain, 1, &kCFTypeArrayCallBacks);
```

Comment:
Whis is this called constKeychain rather than constHaystack?

Reply:
Just a cust & paste thing, the parameter is named `keychain`. I changed it for consistency.